source/tools/testConsistency/ClientApp/app.js

```diff
@@ -0,0 +1,196 @@
+const express = require("express");
```

Comment:
nit: we should have the standard copyright banner here (and in all source files):

https://github.com/microsoft/AdaptiveCards/blob/9b27ee8d79f0010ec246ec736e5722bb54f436f8/source/shared/cpp/ObjectModel/BaseElement.h#L1-L2